src/util/prevStepAPI.js

```diff
@@ -0,0 +1,18 @@
+getPreviousStep = function () {
+    return getStep(-1);
+};
+
+// getNextStep won't exactly be useful since module would have already been run
```

Comment:
Do you mean the next step would not yet have been run? I think it could still be useful for some scenarios, but i agree it's not urgent. Maybe worth it to add for consistency but since we have `getStep()` we should be OK. 

I do think generally we should put these into a file like `stepApi` - more generic, make sense?